core/clock/impl/ticker_impl.hpp

```diff
@@ -29,13 +31,15 @@ namespace kagome::clock {
 
     bool isStarted() override;
 
+    clock::SystemClock::Duration interval() override;
+
     void asyncCallRepeatedly(
         std::function<void(const std::error_code &)> h) override;
 
     void onTick(const boost::system::error_code &ec);
 
    private:
-    bool started_;
+    std::atomic_bool started_;
```

Comment:
Why?

Reply:
there was a case, where ticker was triggered from callback, that possibly was not same thread... Not the case now, so I'll remove it